### PR TITLE
Remove possible contradictions between the GSG and the Cypher manual

### PR DESCRIPTION
**Pull request by NataliaIvakina**
Based on the following Trello card: https://trello.com/c/7kPGMtkE

### CODE REVIEW

File: modules/ROOT/pages/cypher-intro/results.adoc [hunk 1]
```diff
@@ -643,8 +644,9 @@ Rows: 1
 +--------+
 ----
 
-Note that `NULL` values are skipped during aggregation.
-For aggregating only unique values use `DISTINCT`, for example: `count(DISTINCT role)`.
+If you want to skip `null` values, use the function `count(variable)`. 
+
+For aggregating only unique values use the `DISTINCT` operator, for example: `count(DISTINCT role)`.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/cypher-intro/results.adoc:649]: Should these two not have full example queries?

File: modules/ROOT/pages/cypher-intro/results.adoc [hunk 2]
```diff
@@ -671,15 +673,15 @@ Rows: 1
 +--------------------------------------------------------------------------------------------------------------+
 ----
 
-The `count()` function in Cypher allows you to count the number of occurences of entities, relationships, or results returned.
 
-There are two different ways you can count return results from your query.
+Therefore, there are three different ways to use function `count()`:
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/cypher-intro/results.adoc:677]:
```suggestion
There are three different ways to use the `count()` function:
```

File: modules/ROOT/pages/cypher-intro/results.adoc [hunk 3]
```diff
@@ -671,15 +673,15 @@ Rows: 1
...
 
-* The first is by using `count(n)` to count the number of occurences of `n` and does not include `null` values.
+. To count results with `count({empty}*)`, which simply returns the number of matching rows.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/cypher-intro/results.adoc:679]:
```suggestion
. `count({empty}*)`: counts results and returns the number of matching rows.
```

Is `{empty}` clear syntax here? it is not used in Cypher Manual (AFAIK), and Im not really sure what it means?

File: modules/ROOT/pages/cypher-intro/results.adoc [hunk 4]
```diff
@@ -671,15 +673,15 @@ Rows: 1
...
+. To use `count(n)` to count the number of occurences of `n` and to exclude `null` values.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/cypher-intro/results.adoc:680]:
```suggestion
. `count(n)`: count the number of occurrences of `n` (excludes `null` values).
```

File: modules/ROOT/pages/cypher-intro/results.adoc [hunk 5]
```diff
@@ -671,15 +673,15 @@ Rows: 1
...
 You can specify nodes, relationships, or properties within the parentheses for Cypher to count.
-* The second way to count results is with `count({empty}*)`, which counts the number of result rows returned (including those with `null` values).
+. To use the `DISTINCT` operator -- `count(DISTINCT variable)` -- to remove duplicates from the results.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/cypher-intro/results.adoc:682]:
```suggestion
. The `DISTINCT` operator -- `count(DISTINCT variable)` -- removes duplicates from the results.
```

File: modules/ROOT/pages/cypher-intro/results.adoc [hunk 6]
```diff
@@ -671,15 +673,15 @@ Rows: 1
...
 
-In the dataset, some of the `Person` nodes have a Twitter handle, but others do not.
+In the dataset <<people-technologies-companies, _People, technologies, and companies_>>, some of the `Person` nodes have a Twitter handle, but others do not.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/cypher-intro/results.adoc:684]: Is this link useful here? If you click it, it is pretty disruptive to the reading experience. Maybe just, "In the dataset,..." - as it doesnt change on the page?

**Review by JPryce-Aklundh — COMMENTED** [review]
Good work, @NataliaIvakina - I have a few questions, and made a few suggestions.

File: modules/ROOT/pages/cypher-intro/results.adoc (hunk 6 above)
Comment (NataliaIvakina) [reply modules/ROOT/pages/cypher-intro/results.adoc:684]: I think, it's good to have this link here as this piece of content has been clued from two diff. chapters and we use here two datasets: Movie and People, companies.... In the future, I'd like to rewrite all examples to use only one dataset, but in the meantime, it's good to remind users what dataset is used.

File: modules/ROOT/pages/cypher-intro/results.adoc (hunk 1 above)
Comment (NataliaIvakina) [reply modules/ROOT/pages/cypher-intro/results.adoc:649]: IMO, you're right. It's good to illustrate them with code examples. I'll make a note for myself for future

**Pull request merged**